src/middleware/queries.ts

```diff
@@ -212,6 +214,9 @@ export function transactionByHash(
         success
         spec_version_id
         extrinsic_hash
+        block {
+          hash
```

Comment:
Is it ok to not include datetime too here?

Reply:
Yeah. `datetime` param is not used for this query